### ISSUE TITLE
Password authentication fails for Postgres (11) connections

### ISSUE THREAD

**Issue by dspinellis** [simov/express-admin#158]
* Running `node ./node_modules/express-admin/app.js express-admin-examples/config/pg/` errors with `password authentication failed for user "liolio"`. 
* The password and database setup seem to be correct, because providing the same password to `psql -W -h 127.0.0.1 express-admin-examples liolio` succeeds.
* The authentication method used by Postgres seems to be the same in both cases, because providing a wrong password to *psql* causes the connection to fail with exactly the same logged information as that produced by *express-admin*.

```
liolio@express-admin-examples DETAIL:  Password does not match for user "liolio".
        Connection matched pg_hba.conf line 94: "host    all             all             127.0.0.1/32            md5"
```

Could the error stem from an outdated [node-postgres](https://node-postgres.com/) library?  Currently provided [node-posgres connection examples](https://node-postgres.com/features/connecting) don't seem to work with the library version distributed with *express-admin*.  For example, the first code sample fails with `TypeError: Pool is not a constructor`.

**Comment by simov** [comment]
Yes, last time this module was tested was around 2015 so just by looking at the `pg` version on NPM something like https://www.npmjs.com/package/pg/v/4.3.0 should work.

The [documentation](https://simov.github.io/express-admin/) definitely needs an update.

**Comment by dspinellis** [comment]
Thank you for the swift reply! Sadly, running `npm install pg@4.3.0` does not fix the problem.

**Comment by simov** [comment]
Hi again, recently I tested the module again, you can check out my [package.json](https://github.com/simov/express-admin-tests/blob/main/package.json#L18) version for reference. Then I tested with [these two versions](https://github.com/simov/express-admin-tests/blob/main/docker-compose.yml#L32-L49) of PostgreSQL.

**Comment by dspinellis** [comment]
Thank you! Note that I'm no longer trying to use this package, so I'll not be able to provide feedback.
